package.json

```diff
@@ -96,14 +96,14 @@
     "husky": "^3.0.9",
     "jest": "^24.9.0",
     "jest-fetch-mock": "^2.1.2",
-    "jest-styled-components": "^6.3.3",
+    "jest-styled-components": "^6.3.4",
     "json5": "^2.1.1",
     "lerna": "^3.18.4",
-    "lint-staged": "^9.4.2",
+    "lint-staged": "^9.4.3",
```

Comment:
Version bumped for security issue in dependency of `lint-staged`: https://github.com/okonet/lint-staged/releases/tag/v9.4.3